docs/docs/how-to/cloud/working-with-redirects-and-rewrites.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Working with Redirects and Rewrites"
+description: "Learn how to leverage redirects, rewrites, and reverse proxies in Gatsby Cloud"
+label: "cloud"
+---
+
+## What is a redirect?
+
+Redirects are settings in the network layer that allow you to route traffic from one url path to another with little to no disruption.
+
+For instance, while rebuilding your cooking blog, you might want to move all of your recipes from their old path of blog/recipes to a new path of recipes. To make sure that all the existing links to your recipes still work, you would need a way to redirect your users from blog/recipes/mouthwatering-lasagna to recipes/mouthwatering-lasagna. No one wants to lose access to such a, well, mouthwatering recipe.
+
+## ♻️ How to create redirects in Gatsby Hosting
+
+In order to use redirects, you must include the gatsby-plugin-gatsby-cloud in your project.
+
+If your Gatsby project doesn't already have a gatsby-node.js file, add one at that top level of your project (alongside your gatsby-config.js).
+
+In gatsby-node.js file, you'll want to export the createPages method and use the createRedirects action to generate any redirects that you want to add. Here's an example showing the lasagna recipe above:
+
+```javascript:title=gatsby-node.js
+// gatsby-node.js
+exports.createPages = async ({ graphql, actions }) => {
```

Comment:
Indentation/Formatting is off here and in the other code blocks

Reply:
This should be fixed now.

Reply:
(I also ran prettier over the docs)

Reply:
Prettier doesn't run inside codeblocks when a `:title` is used as it can't understand which language it is. As you see below the formatting is still off.

To have it working properly it needs to be

````
```javascript title=gatsby-node.js
````

(But that's just a btw)